propagator/opentelemetry-propagator-jaeger/setup.cfg

```diff
@@ -40,7 +40,7 @@ package_dir=
     =src
 packages=find_namespace:
 install_requires =
-    opentelemetry-api == 1.4.0.dev0
+    opentelemetry-api ~= 1.3
```

Comment:
Is this change only for packages upon release? Packages in development will still pin exact versions right?

Reply:
The plan is to unpin packages in development as well. If a dev package needs a new version of the api, then the dependency will need to be updated for that package at that time.